static/sass/_patterns_pagination.scss

```diff
@@ -22,4 +22,9 @@
   .p-pagination__link--previous {
     margin-inline-end: .5rem;
   }
+
+
+  .p-pagination__item--right {
```

Comment:
Could you try and use the existing alignment utilities instead of adding a new pattern please. https://vanillaframework.io/docs/utilities/align

Reply:
I was looking for something like this. Seems like I was looking at the wrong places. :sweat_smile: